src/main/java/games/strategy/triplea/ui/PlacePanel.java

```diff
@@ -88,9 +88,7 @@ public void territorySelected(final Territory territory, final MouseDetails e) {
       final UnitChooser chooser = new UnitChooser(units, Collections.emptyMap(), getData(),
           false, getMap().getUIContext());
       final String messageText = "Place units in " + territory.getName();
-      if (maxUnits[0] > 0) {
-        chooser.setMaxAndShowMaxButton(maxUnits[0]);
-      }
+      chooser.setMaxAndShowMaxButton(maxUnits[0]);
```

Comment:
Since I'm currently in bed, and this is a very minor change, feel free to edit this snippet in the github online editor.